data/xp-pen-ack05-remote.tablet

```diff
@@ -0,0 +1,35 @@
+# XP-Pen
+# ACK05 Remote
+#
+# Reports as UGTABLET Artist Pro 16 (Gen2), for recordings
+# etc. see https://gitlab.freedesktop.org/libevdev/udev-hid-bpf/-/issues/32
+#
+#  ┌────────┐────────────────────────┐
+# /  ┌────┐  \   ┌───┐┌───┐┌───┐┌───┐│
+# │  │Dial│  │   │ A ││ B ││ C ││   ││
+# │  └────┘  │   └───┘└───┘└───┘│ G ││
+# │\________/    ┌───┐┌───┐┌───┐│   ││
+# │              │ D ││ E ││ F ││   ││
+# │              └───┘└───┘└───┘└───┘│
+# │              ┌───┐┌────────┐┌───┐│
+# │              │ H ││    I   ││ J ││
+# │              └───┘└────────┘└───┘│
+# └──────────────────────────────────┘
+
+[Device]
+Name=UGTABLET Artist Pro 16 (Gen2)
+ModelName=
+DeviceMatch=usb|28bd|095b
+Layout=xp-pen-ack05-remote.svg
+Class=Remote
+
+[Features]
+Stylus=false
+NumDials=1
+NumRings=0
+NumStrips=0
+StatusLEDs=Ring
+
+[Buttons]
+Left=A;B;C;D;E;F;G;H;I;J;
+EvdevCodes=BTN_0;BTN_1;BTN_2;BTN_3;BTN_4;BTN_5;BTN_6;BTN_7;BTN_8;BTN_9;
```

Comment:
```suggestion
# XP-Pen
# ACK05 Remote
#
# Reports as Hanvon Ugee Shortcut Remote, for recordings
# etc. see https://gitlab.freedesktop.org/libevdev/udev-hid-bpf/-/issues/32
#
#  ┌────────┐────────────────────────┐
# /  ┌────┐  \   ┌───┐┌───┐┌───┐┌───┐│
# │  │ A  │  │   │ B ││ C ││ D ││   ││
# │  └────┘  │   └───┘└───┘└───┘│ H ││
# \_________/    ┌───┐┌───┐┌───┐│   ││
#  │             │ E ││ F ││ G ││   ││
#  │             └───┘└───┘└───┘└───┘│
#  │             ┌───┐┌────────┐┌───┐│
#  │             │ I ││    J   ││ K ││
#  │             └───┘└────────┘└───┘│
#  └─────────────────────────────────┘

[Device]
Name=Hanvon Ugee Shortcut Remote
ModelName=
DeviceMatch=usb:28bd:0202
Layout=xp-pen-ack05-remote.svg
Class=Remote

[Features]
Stylus=false
NumDials=1
NumRings=1
NumStrips=0
StatusLEDs=Ring

[Buttons]
Left=A;B;C;D;E;F;G;H;I;J;K;
EvdevCodes=BTN_LEFT;BTN_0;BTN_1;BTN_2;BTN_3;BTN_4;BTN_5;BTN_6;BTN_7;BTN_8;BTN_9;

#Note that no LEDs light up on the device itself
RingNumModes=4
Ring=A
```

Small changes to make my system happier:
- `DeviceMatch seems to be using `:`, not `|`
- the ring button is going to be `A`, and everything gets offset by 1
- support for ring modes?

Libwacom is happy, but if I take @Deevad SVG, the panel doesn't show the svg, so I guess something is off....

Reply:
> but if I take @Deevad SVG, the panel doesn't show the svg, so I guess something is off....

Hey, yes, I'm a bit stuck with the step where I have to run clean_svg.py script to autorename/autoclean elements in the SVG as noted [on the documentation](https://github.com/linuxwacom/libwacom/blob/master/data/layouts/README.md#automatic-naming-with-inkscape-and-clean_svgpy). 

Unfortunately, my attempts are failing, I reported it here above: https://github.com/linuxwacom/libwacom/pull/777#issuecomment-2406520060 : the script change the label of the buttons for Group A/ Group B, move the buttons around, delete the frame of the device, etc...